src/pages/_app.tsx

```diff
@@ -75,7 +80,7 @@ const handleNetworkModeQueryParam = (store: EnhancedStore, appContext: AppContex
   console.log('[debug] activeNetwork', activeNetwork);
   console.log('[debug] networks', networks);
   const queryNetworkMode = ((Array.isArray(query.chain) ? query.chain[0] : query.chain) ||
-    '') as NetworkModes;
+    NetworkModes.Mainnet) as NetworkModes;
```

Comment:
I've never seen an error with this in prod, but locally I was encountering issues with this being an empty string and the network mode being set to `undefined`. Is Mainnet a reasonable default here?

Reply:
yep! I fixes this in my latest PR